command/host_create.go

```diff
@@ -33,11 +34,9 @@ func (c *createHost) ParseParams(params map[string]interface{}) error {
 		c.CreateHost.Background = true
 	}
 
-	// if a filename is defined, get the params from the file
 	if _, ok := params["file"]; ok {
 		fileName := fmt.Sprintf("%v", params["file"])
-		return errors.Wrapf(utility.ReadYAMLFile(fileName, &c.CreateHost),
-			"error reading from file '%s'", fileName)
+		c.File = fileName
```

Comment:
should we validate the user hasn't both passed a file and defined the parameters?

Reply:
fixed. Thanks!